examples/wnd_outbrain/Outbrain.ipynb

```diff
@@ -439,91 +425,46 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "We can save the stats from the workflow and load it anytime, so we can run training without doing preprocessing."
+    "We select certain categorical and numerical features that are processed and generated via the NVTabular workflow to train our W&D TF model. Below, we'll create NUMERIC_COLUMNS list to feed our WnD model."
```

Comment:
I think we should be consistent with our abbreviations - we should either use W&D or WnD, but not both. 
```suggestion
    "We select certain categorical and numerical features that are processed and generated via the NVTabular workflow to train our W&D TF model. Below, we'll create NUMERIC_COLUMNS list to feed our W&D model."
```